compiler-builtins/src/mem/impls.rs

```diff
@@ -384,13 +384,13 @@ pub unsafe fn set_bytes(mut s: *mut u8, c: u8, mut n: usize) {
 }
 
 #[inline(always)]
-pub unsafe fn compare_bytes(s1: *const u8, s2: *const u8, n: usize) -> i32 {
+pub unsafe fn compare_bytes(s1: *const u8, s2: *const u8, n: usize) -> crate::mem::c_int {
```

Comment:
Could you import `core::ffi::c_int` and use `c_int` here instead?

I didn't even realize we had `crate::mem::c_int`, I'll probably remove that to just use `core::ffi` instead.

Reply:
> I didn't even realize we had `crate::mem::c_int`, I'll probably remove that to just use `core::ffi` instead.

Doing this here https://github.com/rust-lang/compiler-builtins/pull/981